package.json

```diff
@@ -59,7 +59,7 @@
     "use-query-params": "^1.1.9",
     "yup": "~0.29.3"
   },
-  "proxy": "http://localhost:4000",
+  "proxy": "http://localhost:4100",
```

Comment:
Not sure why this change is needed from 4000 to 4100

Reply:
4100 is the address of the api gateway